File: app/models/flight.rb
# == Schema Information
#
# Table name: flights
#
#  id           :integer          not null, primary key
#  aeroplane_id :integer
#  origin       :string
#  destination  :string
#  date         :date
#  created_at   :datetime         not null
#  updated_at   :datetime         not null
#

class Flight < ActiveRecord::Base
  belongs_to :aeroplane
  has_many :reservations
end

File: app/assets/javascripts/models/Flight.js
var app = app || {};

app.Flight = Backbone.Model.extend({
  urlRoot: '/flights'
});


// create_table "flights", force: :cascade do |t|
//   t.integer  "aeroplane_id"
//   t.string   "origin"
//   t.string   "destination"
//   t.date     "date"
//   t.datetime "created_at",   null: false
//   t.datetime "updated_at",   null: false
// end
